scarlet/observation.py

```diff
@@ -480,12 +555,23 @@ def _render(self, model):
             The convolved and resampled `model` in the observation frame.
         """
         model_ = model[self._band_slice,:,:]
-        model_ = np.array([np.dot(model_[c].flatten(), self._resconv_op[c]) for c in range(self.frame.C)], dtype=self.frame.dtype)
+        model_image = []
+        for c in range(self.frame.C):
+
+            model_conv1d = sinc_shift_1D(model_[c], self._coord_hr[self.small_axis], self.small_axis, self.obs_fast)
+            model_shape = np.shape(model_conv1d)
+            # I dont KNOW why this stupid axis is inverted and it KILLS ME!!!!
```

Comment:
Could this have something to do with the PSF coordinate inversion in https://github.com/pmelchior/scarlet/blob/fast_resampling/scarlet/observation.py#L404

Reply:
Hmm, that is strange.